squad/ci/backend/lava.py

```diff
@@ -45,7 +45,11 @@ def fetch(self, test_job):
             data['results'] = yaml.load(yamldata)
 
             # fetch logs
-            logs = self.__get_job_logs__(test_job.job_id)
+            logs = ""
+            try:
+                logs = self.__get_job_logs__(test_job.job_id)
+            except:
```

Comment:
I was getting TypeError. The idea here is to catch any errors so problems fetching logs doesn't prevent from getting the results. I can also do that in lava backend and create a BackendException to encapsulate the problem.